### ISSUE TITLE
Add failing pages for testing

### ISSUE THREAD

**Issue by nschonni** [squizlabs/HTML_CodeSniffer#63]
Adding individual pages that fail the rules would make creating and testing PR's easier to verify

**Comment by ironikart** [comment]
Test scripts are working and have been expanded to include additional use cases. Check out the `Tests/WCAG2/` directory for example html. The test runner is a bash script at `Contrib/Testing/run_tests.sh` that uses PhantomJS. The tests don't have full coverage, very much a work in progress but they should give some basic assurance for pull requests.
